controllers/metrics/metrics.go

```diff
@@ -50,34 +50,46 @@ var (
 		Help:      "requests to list content revisions on grafana.com",
 	}, []string{"kind", "resource", "method", "status"})
 
+	InitialStatusSyncDuration = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: "grafana_operator",
+		Subsystem: "reconciler",
+		Name:      "initial_sync_duration",
+		Help:      "time in ms to sync statuses after operator restart",
+	})
+
+	// Deprecated: All Initial Sync Duration metrics have merged into a single metric
```

Comment:
Do we use these metrics in the provided dashboard? If so, we should adapt this as well